app/adapters/comment.js

```diff
@@ -0,0 +1,22 @@
+import ApplicationAdapter from './application';
+
+export default ApplicationAdapter.extend({
+  pathForType() {
+    return 'articles';
+  },
+
+  urlForCreateRecord(modelName, snapshot) {
```

Comment:
Does it make sense and is it possible to use `buildUrl` here for all three of these?
Something like:
```js
buildURL(type, id, record) {
        const namespace = get(this, "namespace");
        return this._super(type, id, record).replace(namespace, `${namespace}/${record.get("article.id")}/comments/${id}`);
    },
```

Reply:
Yes, there's a pattern to these "urlFor" methods that can be addressed with `buildURL`.

Reply:
Added the `buildURL` and removed the others.